analyzer/throttle.c

```diff
@@ -61,15 +61,15 @@ suscan_throttle_get_portion(suscan_throttle_t *throttle, SUSCOUNT h)
   if (h > 0) {
     do {
       retry = SU_FALSE;
-      clock_gettime(CLOCK_MONOTONIC, &tn);
+      tn = suscan_gettime_raw();
```

Comment:
I've just done some additional testing (Linux box) and, somehow, using `suscan_gettime_raw` here keeps the actual processing rate always below the desired rate, while using `suscan_gettime` makes it oscillate above and below the desired rate. Are you experiencing this behavior in FreeBSD too?

Reply:
I see a little bit of movement, but I assumed that was reasonable given the non-realitime architecture.
For example, for a 1.024 MS/s replay, I see the processing rate fluctuate between about 1.022 and 1.026 MS/s.  But most of the time, SigDigger displays 1.024 MS/s rate. Is there a better way to detect any oscillation than just eyeballing the GUI?

Reply:
I should add, that it looks like after a long series of 1.024, it drops down to 1.022 for about a second, then to 1.026 for a second, and then back to 1.024 for many seconds.  (I'm guessing the UI updates only once per second...)

Reply:
That's with `CLOCK_MONOTONIC` being used by both `suscan_gettime` and `suscan_gettime_raw` since FreeBSD doesn't have `_RAW`.